lasagne/nonlinearities.py

```diff
@@ -83,29 +83,56 @@ def rectify(x):
 
 # leaky rectify
 class LeakyRectify(object):
-    """Implementation of a leaky rectifier
-    :math:`\\varphi(x) = \max(leakiness \cdot x, x)`.
+    """Leaky rectifier :math:`\\varphi(x) = \\max(\\alpha \\cdot x, x)`
+
+    The leaky rectifier was introduced in [1]_. Compared to the standard
+    rectifier :func:`rectify`, it has a nonzero gradient for negative input,
+    which often helps convergence.
 
     Parameters
     ----------
-    leakiness : float in [0, 1]
+    leakiness : float
+        Slope for negative input, may be chosen arbitrarily.
         A leakiness of 0 will lead to the standard rectifier,
-        a leakiness of 1 will lead to a linear activation function.
+        a leakiness of 1 will lead to a linear activation function,
+        and any value in between will give a leaky rectifier.
 
     Methods
     -------
     __call__(x)
-        Calculate the neuron's output by applying this leaky rectifier to the
-        activation `x`.
+        Calculate the neuron's output for activation `x`.
```

Comment:
It shouldn't mention neurons at all, I think. I'll change it to something more neutral.